website/docs/r/org.html.markdown

```diff
@@ -9,19 +9,26 @@ Provides a vCloud Director Organization resource. This can be used to create and
 # vcd\_org
 
 Provides a vCloud Director Org resource. This can be used to create and delete an organization.
+Requires administrator privileges.
```

Comment:
For clarity I would use "system administrator", as org admin is also administrator.

Reply:
done